### DISCUSSION TITLE
Recurisve search for clips.

### DISCUSSION THREAD

**Discussion by ngysling** [Hubro/clipface#21] [Ideas]
I think a great addition to Clipface would be to have the program recursively search through all directories from the root (/clips) and add clips from there. Right now, my clips folder has different sub-directories that separate the topics of the clips. Having the directory structure in the interface would be nice to make finding a specific clip easier. Another suggestion of mine would be the customization of the theme of the webpage (i.e dark mode) and the ability to personalize the "Clippy McClipFace" text at the top.

**Comment by Hubro** [comment]
- Recursively search for clips - I will definitely add this
- Dark mode - There's an issue open for this, I'll probably add this soon
- Customize the page title - Yeah that's probably a good idea. Should be relatively quick to fix.

Thanks for the feedback 🙂

**Reply by ngysling** [reply]
Thank you for the quick response and for creating this! Finding this was the first time I actively got excited to set something up in a while lol.

**Comment by f-dav** [comment]
> 
> 
> ...... and the ability to personalize the "Clippy McClipFace" text at the top.

I just finished this change (also logged as Issue https://github.com/Hubro/clipface/issues/23), and Hubro has merged it. The docker image still needs an update though, to reflect the change :)
